### PR TITLE
Add support for v1beta1 device plugin API.

### PR DESCRIPTION
**Pull request by jiayingz**
Other than unit tests, also ran GPUDevicePlugin e2e test with and without PR https://github.com/kubernetes/kubernetes/pull/58282 to make sure the gke gpu device plugin works with both v1alpha kubelet and v1beta1 kubelet.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -37,8 +37,10 @@ presubmit: vet
 TAG?=$(shell git rev-parse HEAD)
 REGISTRY?=gcr.io/google-containers
 IMAGE=nvidia-gpu-device-plugin
+REPO="github.com/GoogleCloudPlatform/container-engine-accelerators"
 
 build:
+	go install ${REPO}/pkg/gpu/nvidia
```
Comment (vishh) [inline Makefile]: You don't need to install

File: Dockerfile [hunk 2]
```diff
@@ -15,6 +15,7 @@
 FROM golang:1.9-alpine as builder
 WORKDIR /go/src/github.com/GoogleCloudPlatform/container-engine-accelerators
 COPY . .
+RUN go install github.com/GoogleCloudPlatform/container-engine-accelerators/pkg/gpu/nvidia
```
Comment (vishh) [inline Dockerfile]: Why is this necessary?

**Comment by vishh** [comment]
Fix the `use_alpha` logic as discussed offline and then this is good to be merged. Feel free to self-merge.

File: pkg/gpu/nvidia/manager.go [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+	nvidiaCtlDevice = "/dev/nvidiactl"
+	nvidiaUVMDevice = "/dev/nvidia-uvm"
+	// Optional device.
+	nvidiaUVMToolsDevice = "/dev/nvidia-uvm-tools"
+	devDirectory         = "/dev"
+	nvidiaDeviceRE       = `^nvidia[0-9]*$`
+)
+
+var (
+	hostPathPrefix      = flag.String("host-path", "/home/kubernetes/bin/nvidia", "Path on the host that contains nvidia libraries. This will be mounted inside the container as '-container-path'")
```
Comment (vishh) [inline pkg/gpu/nvidia/manager.go]: nit: Flags should be in injected to libraries.

**Comment by vishh** [comment]
LGTM

**Comment by jiayingz** [comment]
Thanks a lot for the quick review!
